src/Microsoft.ML.CpuMath/Sse.cs

```diff
@@ -1474,44 +1320,43 @@ public static void ZeroMatrixItems(AlignedArray dst, int ccol, int cfltRow, int[
             }
         }
 
-        public static void SdcaL1UpdateDense(float primalUpdate, int length, float[] src, float threshold, float[] v, float[] w)
+        public static void SdcaL1UpdateDense(float primalUpdate, int count, ReadOnlySpan<float> src, float threshold, Span<float> v, Span<float> w)
```

Comment:
>count [](start = 69, length = 5)

this is a somewhat conspicuous rename, because `length` and `count` are both used in a very specific way in our codebase.

Maybe it's worthwhile to add a summary comment to this method to denote exactly what is the expectation of `count` ?

Reply:
In the `ML.CpuMath` assembly, all the code uses the term `count` to mean "how many elements to work over". This was the only externally visible usage of `length`, so I decided to rename it to be consistent with the rest of the classes in `ML.CpuMath`. It also fixes the issue with `SdcaL1UpdateSparse` that `length` wasn't used at all, but the other parameter `count` WAS used. Now both `SdcaL1UpdateDense` and `SdcaL1UpdateSparse` have a consistent signature, with the only difference that Sparse takes in the `indices` as well.